integration flake: TestPersistentVolumeMultiPV

```
=== RUN   TestPersistentVolumeMultiPVs
--- FAIL: TestPersistentVolumeMultiPVs (10.96s)
	persistent_volumes_test.go:254: volumes created
	persistent_volumes_test.go:260: claim created
	persistent_volumes_test.go:264: volume bound
	persistent_volumes_test.go:266: claim bound
	persistent_volumes_test.go:284: Bind mismatch! Expected pvc-2 capacity 50000000000 but got fake-pvc-57 capacity 5000000000
```

https://pantheon.corp.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/27176/kubernetes-pull-test-unit-integration/30016/artifacts/

@jsafrane

--- a/test/integration/persistent_volumes_test.go
+++ b/test/integration/persistent_volumes_test.go
@@ -250,6 +250,7 @@ func TestPersistentVolumeMultiPVs(t *testing.T) {
 		if err != nil {
 			t.Errorf("Failed to create PersistentVolume %d: %v", i, err)
 		}
+		waitForPersistentVolumePhase(testClient, pvs[i].Name, watchPV, api.VolumeAvailable)
 	}
 	t.Log("volumes created")
 

